experimental/ast/decl_file.go

```diff
@@ -70,6 +72,38 @@ func (f File) Imports() iter.Seq2[int, DeclImport] {
 	}
 }
 
+// ToProtoOptions contains options for the [File.ToProto] function.
+type ToProtoOptions struct {
+	// If set, no spans will be serialized.
+	//
+	// This operation only destroys non-semantic information.
+	ElideSpans bool
```

Comment:
nit: I typically see these sorts of flags named using the word `Omit` instead of `Elide`. So maybe rename to `OmitSpans` and `OmitFile`.